content/api/commands/attachfile.md

```diff
@@ -0,0 +1,217 @@
+| `waitForAnimations` | [`waitForAnimations`](/guides/references/configuration#Actionability) | Whether to wait for elements to [finish animating](/guides/core-concepts/interacting-with-elements#Animations) before executing the command.                        |
+
+### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
+
+<List><li>`.attachFile()` yields the same subject it was given from the previous command.</li></List>
+
+### Action
+
+Depending on the action set in the `options` argument, `.attachFile()` can simulate two different user behaviors:
+
+- `input` (default)
+By default, `attachFile()` runs in 'input' mode, mimicking a user selecting one or more files on an HTML5 input element. In this mode, the subject [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management) must be a single `input` element with `type="file"`, or a `label` element connected to an input (either with its `for` attribute or by containing the input).
+- `drag-n-drop`
+Setting the action to `drag-n-drop` changes the behavior of the command to instead mimic a user dragging files from the operating system into the browser, and dropping them over the selected subject. In this mode, the subject can be any DOM element or the `document` as a whole.
+
+## Examples
+
+### From a file on disk
+
+```javascript
+cy.get('input[type=file]').attachFile('file.json')
+cy.get('input[type=file]').attachFile('file.png')
+```
+
+If given a path, `.attachFile()` will attach the file exactly as it exists on disk. This is the preferred way to work with files on disk, avoiding many encoding-related pitfalls.
+
+### On a hidden input
+
+```javascript
+cy.get('input[type=file]').attachFile('file.json', { force: true })
+```
+
+In many cases in modern applications, the underlying file input is hidden from view, and activated by a user clicking on a button. In these cases, you will need to tell Cypress to ignore its actionability checks and attach the file even though a user would not be able to directly activate the file input.
+
+### From a fixture
+```javascript
+cy.fixture('file.json', { encoding: null }).as('myFixture')
+cy.get('input[type=file]').attachFile('@myFixture')
+```
+
+Note the use of `null` encoding. By default, `cy.fixture()` and `cy.readFile()` attempt to interpret files read from disk, which would result in a JSON file being decoded and re-encoded as a utf-8 string - the contents would be preserved, but formatting would not be and the encoding might change. See [`cy.fixture`](/api/commands/fixture) or [`cy.readFile`](/api/commands/readfile) for more details on file encoding.
+
+### From an API response
+
+```javascript
+cy.request('http://localhost:8888/users/827').get('body').as('responseBody')
```

Comment:
This isn't correct. `cy.get()` cannot be chained off of another command. It's a parent command.

Reply:
You're right, the example is incorrect. Oddly though, it doesn't throw an error:
```
cy.request('http://example.com').get('body').then(a => { console.log(a) })
// a is now the <body> element of the AUT
```
Maybe something to consider fixing with `.get()` at some point, making it stricter.

Will fix the example.

Reply:
Yah, we never got around to enforced chaining. That would be a major breaking change.